micrometer-core/src/main/java/io/micrometer/core/instrument/config/MeterRegistryConfigValidator.java

```diff
@@ -67,8 +68,8 @@ public static <M extends MeterRegistryConfig, T> Function<M, Validated<T>> check
         };
     }
 
-    public static <M extends MeterRegistryConfig, T> Function<M, Validated<T>> checkRequired(String property,
-            Function<M, T> getter) {
+    public static <M extends MeterRegistryConfig, T extends @Nullable Object> Function<M, Validated<T>> checkRequired(
```

Comment:
This was an attempt to get rid of errors like the following:

```
implementations/micrometer-registry-azure-monitor/src/main/java/io/micrometer/azuremonitor/AzureMonitorConfig.java:72: error: [NullAway] referenced method returns @Nullable, but functional interface method java.util.function.Function.apply(T) returns @NonNull
                check("connectionString", AzureMonitorConfig::connectionString));
                                          ^
    (see http://t.uber.com/nullaway )
```

Some config methods will have a nullable return value; some won't. How do we mark that accordingly, or is this a limitation of what NullAway can understand?